pkg/machinery/constants/constants.go

```diff
@@ -817,3 +817,9 @@ var Overlays = []string{
 	"/usr/etc/udev",
 	"/opt",
 }
+
+// DefaultDroppedCapabilities is the default set of capabilities to drop.
+var DefaultDroppedCapabilities = []string{
```

Comment:
we could make this `map[string]struct{}` for easier checks?

Reply:
I didn't quite get that

Reply:
I mean if we change it from being a slice to being a map with capabilities as keys, we can simplify lookup

Reply:
I mean https://github.com/siderolabs/talos/pull/6822/files#diff-0be4290a97f76e2ccbaa28623fbfd909d32b6385dabe780a6acf080af7025cd2R24 here

Reply:
fixed